docs/docs/guides/troubleshooting.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Troubleshooting Neosync Jobs
+id: troubleshooting
+hide_title: false
+slug: /guides/troubleshooting
+---
+
+## Introduction
+
+Databases are _complex_. Neosync does its best to handle any and all edge cases you throw at it.
+
+This document is a collection of common issues you may face when running Neosync and how to resolve them.
+
+### `Sync: could not successfully complete sync activity: read tcp 0.0.0.0:42454->0.0.0.0:5432: i/o timeout`
+
+This is likely due to an overabundance of connections against the database. 
+
+You can resolve this by reducing the number of open connections in the "Connections" tab of Neosync. The default is at 80, the easiest way to debug is to start with something small like 10 and increase it until you hit the error again.
+
+### `could not successfully complete sync activity: pq: insert or update on table \"table_name\" violates foreign key constraint \"table_name_id_fkey\"`
```

Comment:
Same here, this is valid given the context.

```
# cSpell:words fkey
```

Reply:
I swapped this one to `foreign_key` seems likely that someone might add "fkey" accidentally else where in the file 👍🏻

Reply:
sounds good 👍